src/main/js/page_objects/blueocean/bluePipelineActivity.js

```diff
@@ -35,5 +36,16 @@ module.exports.commands = [{
         this.waitForElementVisible('.activity-table tr#' + runName);
         this.waitForElementVisible('.activity-table tr#' + runName + ' svg.svgResultStatus');
         this.waitForElementPresent('.activity-table tr#' + runName + ' svg path.running');
+    },
+    clickTab: function (browser, tab) {
+        var self = this;
+        return url.clickTab(browser, self, tab);
+    },
+    assertActivitiesToBeEqual: function (browser, expected) {
+        var self = this;
+        self.waitForElementVisible('@activityTableEntries');
+        browser.elements('css selector', 'table.activity-table tbody tr', function (codeCollection) {
```

Comment:
Can these selectors not be defined on the page=object and then `@` referenced using the nightwatch selenium wrapper functions?